foundry.toml

```diff
@@ -0,0 +1,3 @@
+[profile.default]
```

Comment:
I think we should set more parameters like solc version

Reply:
Yes np, but I think it should be especially if we're doing gas snapshot to compare

Reply:
We are not doing this here. So should I do it ? If yes, with which version ?

Reply:
Not sure to understand why, but ok why not.

Reply:
I thought you wanted to add gas snapshot on this repo. If not then you can forget about it